connect-a-go-app-to-cockroachdb.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Connect a Go App to CockroachDB
+summary: Learn how to connect a simple Go application to a CockroachDB cluster using either a low-level client driver or an ORM.
```

Comment:
s/Learn how to connect a simple Go application to a CockroachDB cluster/Learn how to use CockroachDB from a simple Go application/